src/elf/parser.rs

```diff
@@ -88,7 +88,7 @@ fn get_elf_section_name<'a>(sect: &Elf64_Shdr, strtab: &'a [u8]) -> Option<&'a s
     extract_string(strtab, sect.sh_name as usize)
 }
 
-struct Elf64ParserBack {
+struct ElfParserBack {
     ehdr: Option<Elf64_Ehdr>,
     shdrs: Option<Vec<Elf64_Shdr>>,
```

Comment:
I think it does deal only with 64-bit ELF files, which is why it's called Elf64Parser. In the future we can start supporting 32-bit ELFs, so maybe let's keep it as is? Unless the idea is to make ElfParser to work with both 32-bit and 64-bit files transparently?

Reply:
I think we should try and handle it transparently, certainly if the intention is to expose that type.

Reply:
Yeah, I think transparently would be easier to develop and maintain, as parts that are not 32/64-specific would be more naturally contained.